models/user.server.model.js

```diff
@@ -1,13 +1,15 @@
-const User = require('../config/db_schemas/user.schema');
+const
+    User = require('../config/db_schemas/user.schema'),
+    saltedMd5 = require('salted-md5');
 
 /**
  * Hashes a given plaintext password
  * @param password plaintext password for hashing
  * @returns {String} hashed password
  */
-function hashPassword(password) {
-    // TODO: Create a function for hashing (and maybe salting) passwords
-    return password;
+hashPassword = function(password) {
```

Comment:
```suggestion
hashSaltPassword = function(plaintextPassword) {
```
If we are going to use a more readable name then why not use hashSaltPassword() and plaintextPassword to more clearly define the state of the password in each case

Reply:
Sounds good to me, done.